.github/workflows/auto_add_pr_to_miletone.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           dotnet-version: '7.0.101'
 
       - name: "Assign to vNext Milestone"
-        run: ./tracer/build.sh AssignPullRequestToMilestone
+        run: .//github-actions-helpers/build.sh AssignPullRequestToMilestone
```

Comment:
`.//` There should really be two `/`? (Idk if it matters)

Reply:
There shouldn't, it's a typo. I had a hard time seeing it even when you pointed it out. Thanks a lot.